src/data/resolvers/extensions.ts

```diff
@@ -0,0 +1,42 @@
+import { Resolvers } from '@apollo/client';
+
+import { ClientType } from '@colony/colony-js';
+
+import { Context } from '~context/index';
+
+export const extensionsResolvers = ({
+  colonyManager,
+}: Required<Context>): Resolvers => ({
+  Query: {
+    async votingExtensionParams(_, { colonyAddress }) {
+      try {
+        const extensionClient = await colonyManager.getClient(
+          ClientType.VotingReputationClient,
+          colonyAddress,
+        );
+
+        const stakePeriodBigNum = await extensionClient.getStakePeriod();
+        const stakePeriod = stakePeriodBigNum.toNumber();
+
+        const submitPeriodBigNum = await extensionClient.getSubmitPeriod();
+        const submitPeriod = submitPeriodBigNum.toNumber();
+
+        const revealPeriodBigNum = await extensionClient.getRevealPeriod();
+        const revealPeriod = revealPeriodBigNum.toNumber();
+
+        const escalationPerBigNum = await extensionClient.getEscalationPeriod();
+        const escalationPeriod = escalationPerBigNum.toNumber();
```

Comment:
As a precaution, this should be converted to strings, as JS is weird when it comes to large numbers.

It works perfectly as-is, I just wanted to mention this so you could be aware